parquet-common/src/main/java/org/apache/parquet/hadoop/metadata/CompressionCodecName.java

```diff
@@ -19,7 +19,6 @@
 package org.apache.parquet.hadoop.metadata;
 
 
-import org.apache.parquet.format.CompressionCodec;
```

Comment:
I am not sure why it got removed. This import is needed in this class and even the checks are failing because of it.